client/my-sites/plans/current-plan/thank-you-header/style.scss

```diff
@@ -0,0 +1,39 @@
+/** @format */
+.thank-you-header {
+	box-shadow: none;
+	text-align: center;
+
+	@include breakpoint( '>800px' ) {
+		font-size: 16px;
+		// padding: 50px 80px 30px;
+		margin-bottom: 40px;
+	}
+	@include breakpoint( '>1280px' ) {
+		padding: 50px 160px 30px;
```

Comment:
Such large left/right paddings usually show that we could have done this with a fixed width container that's horizontally centered.

Reply:
So we'd wrap the internal content in another container with a `max-width`? That seems like a fine alternative. Is there a reason to avoid using padding?

Reply:
Well, let's consider that we want to reuse this and drop it in another container that has max-width or fixed width - then the padding would be unexpected.

I'm good with using padding, it's just that my feeling is that we use padding here to conceal the max width that we have, while we could explicitly specify the max width, which is what our actual goal is.